cypress/e2e/WebInterface/Test Cases/QI-CORE Test Case/TestCaseExportImport/TestCaseImportValidations.cy.ts

```diff
@@ -24,8 +24,8 @@ let measureName = 'TestMeasure' + Date.now()
 let CqlLibraryName = 'TestLibrary' + Date.now()
 let testCaseTitle = 'Passing Test Case'
 let secondTestCaseTitle = 'Failing Test Case'
-let testCaseDescription = 'DENOMPass' + Date.now()
-let secondTestCaseDescription = 'DENOMFail' + Date.now()
+let testCaseDescription = 'DENOMPass'
+let secondTestCaseDescription = 'DENOMFail'
```

Comment:
The variation didn't seem necessary, and I needed static data for some checks I added.